versions.mustache

```diff
@@ -0,0 +1,4 @@
+WordPress version: {{wp-version}}
```

Comment:
Please put this file into a `templates` subfolder to stick with current conventions.